### PR TITLE
add comments route

### PR DESCRIPTION
**Pull request by vbatallones**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 
3
Completeness Level (1-5):
4
What did you think of this deliverable?:
confused

### CODE REVIEW

File: controllers/articles.js [hunk 1]
```diff
@@ -29,15 +29,59 @@ router.get('/new', (req, res) => {
...
     where: { id: req.params.id },
     include: [db.author]
   })
   .then((article) => {
     if (!article) throw Error()
-    console.log(article.author)
-    res.render('articles/show', { article: article })
+    db.comment.findAll({
+      where: {articleId: req.params.id}
+    })
```
Comment (petefitton) [inline controllers/articles.js:58]: This is a way to find the comments that have a particular articleId.  An easier way is to add the comments that are associated with the article when you find that article higher up in this route.  If you see where you include the db.author on line 52, you can also include the db.comment to grab all of the associated comments.  Just want to note this for your own knowledge!  Ex for line 52:

include: [db.author, db.comment]

File: controllers/articles.js [hunk 2]
```diff
@@ -29,15 +29,59 @@ router.get('/new', (req, res) => {
...
+  })
+})
+
+// POST route for comments
+router.post('/comment', (req, res) => {
+  console.log('req.body')
+  console.log(req.body)
+  db.comment.create(
+    req.body
+  )
```
Comment (petefitton) [inline controllers/articles.js:81]: I think Adam talked about using req.body in class yesterday.  How did this work for you?  I always use the more explicit syntax, but from what I remember, this should work!

**Comment by petefitton** [comment]
Levin, it looks like you did a really good job!  Do you have particular concepts or parts of this that stick out for you as being difficult?  Let us know if you have any questions - definitely want you to get clarification on parts that may still be confusing!

**Comment by vbatallones** [comment]
Got it. Thank you Pete, I will make those changes

**Pull request closed without merging**